### ISSUE TITLE
Feature request: configuration option to disable update-checker

### ISSUE THREAD

**Issue by SlimeDog** [tr7zw/Item-NBT-API#96]
Spigot 1.16.1
NBTAPI 2.4.1 and earlier

Please implement configuration option to disable update-checker. We have our own automagical update-checker. We block calls by plugins at the network, which results in
```
[11:41:17] [Thread-50/WARN]: [NBTAPI] Error while checking for updates!
java.io.IOException: Connect failed
        at me.lucko.networkinterceptor.interceptors.ProxySelectorInterceptor$LoggingSelector.select(ProxySelectorInterceptor.java:59) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1181) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1071) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$6.run(HttpURLConnection.java:1069) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1068) ~[?:?]
        at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1592) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1512) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection$9.run(HttpURLConnection.java:1510) ~[?:?]
        at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
        at java.security.AccessController.doPrivilegedWithCombiner(AccessController.java:795) ~[?:?]
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1509) ~[?:?]
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250) ~[?:?]
        at de.tr7zw.nbtapi.utils.VersionChecker.checkForUpdates(VersionChecker.java:34) ~[?:?]
        at de.tr7zw.nbtapi.utils.MinecraftVersion.lambda$init$0(MinecraftVersion.java:85) ~[?:?]
        at java.lang.Thread.run(Thread.java:834) [?:?]
```
Thank you.

**Comment by tr7zw** [comment]
Is in the newest SNAPSHOT 2.5.0-SNAPSHOT, not yet in the version on spigotmc.

**Comment by SlimeDog** [comment]
I don't find a link for the dev/snapshot releases. I would be happy to test them.

**Comment by SlimeDog** [comment]
Confirmed fixed in 2.5.0-b169.
